expansions/pandemonium/readme.md

```diff
@@ -4,7 +4,7 @@ Pandemonium
 A WIP Requiem expansion, upgrading soul capabilities.
 
 
-###Changes to Requiem
+### Changes to Requiem
```

Comment:
Is this a standard unicode space (U+0020) ? We need to make sure we don't pollute the codebase with exotic spaces like U+202F.

Reply:
After discussion on Discord, it seems that this space is conform to our standards. I still advise you to be more rigorous in your testing for further contributions.
Edit: Alright, I double checked with some Markdown syntax analysis software, this space is definitely conform.